miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/StorageLocationRestController.java

```diff
@@ -58,13 +60,13 @@
         .collect(Collectors.toList());
   }
 
-  @RequestMapping(value = "/freezers", method = RequestMethod.POST)
+  @PostMapping(value = "/freezers")
   @ResponseStatus(HttpStatus.CREATED)
   public @ResponseBody StorageLocationDto createFreezer(@RequestBody StorageLocationDto dto) {
     return doSave(dto);
   }
 
-  @RequestMapping(value = "/freezers/{id}", method = RequestMethod.PUT)
+  @PutMapping(value = "/freezers/{id}")
   public @ResponseBody StorageLocationDto update(@PathVariable(name = "id", required = true) long id, @RequestBody StorageLocationDto dto)
       throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)